front-end/src/components/card/card.jsx

```diff
@@ -2,21 +2,57 @@ import React from 'react';
 import Button from '../buttons/button';
 import './card.css';
 
+const product = 'http://localhost:8080/product';
+
 //Create a card component for products, should include Title, description, price,
 // add to cart button
 class Card extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            items: [],
+            isLoaded: false,
+        }
+    }
+
+    componentDidMount() {
+        fetch(product)
+            .then(response => response.json())
+            .then(json => 
+                this.setState({
+                    isLoaded: true,
+                    items: json,
+                }));
+    }
+
     render() {
-        return (
-            <div className='card'>
-                <h1 className='title'>Title</h1>
-                <h1 id='description'>Description</h1>
-                <h1 id='price'>Price</h1>
-                <div className='addToCart'>
-                    <Button />
-                </div>
-            </div >
+        var { isLoaded, items } = this.state;
+        if (!isLoaded) {
+            return <div>Loading...</div>;
+        }
+        else {
+            return (
+                <div className='card'>
+                    {
+                        <ul>
+                            {items.map(item =>
+                                <li key={item.id}>
+                                    <h1 className='name'>  Name: {item.name} </h1>
```

Comment:
This could be a Card component all the h1 and h2 and h3 and just pass props to it. this way you will understand how props work.